### ISSUE TITLE
Consider Proxy Settings

### ISSUE THREAD

**Issue by jeffsnow88** [PocketMobsters/PokeMobBot#233]
Hello Everyone, Any possibility to implement proxy configuration? I'm running behind a proxy, so the bot can't connect as no configuration is set.

Ty.

**Comment by Murumuru** [comment]
It would be a nice thing to have.

@jeffsnow88 
You probably can use "Proxifier" for now.

**Comment by zno3** [comment]
agree, i use proxifier when ptc server on my region down, and use another region instead, proxy pretty useful i think

**Comment by chauhan-rahul** [comment]
I am also behind a proxy kindly implement this functionality !
